src/components/reference/_pro-rules-language-coverage.mdx

```diff
@@ -0,0 +1,11 @@
+<details>
+    <summary>Click to view languages with Pro rules coverage</summary>
```

Comment:
I don't have strong opinions on this, but maybe it would be helpful to have these alphabetized?

Reply:
Nice catch, I just followed the list without considering that.